### DISCUSSION TITLE
Chapter 1 - Molecular Dynamics

### DISCUSSION THREAD

**Discussion by argunaykut** [softmatterlab/SOCS#7] [General]
Here, you can ask questions and discuss issues about the simulations in Chapter 1 - Molecular Dynamics.
We encourage you to participate in these discussions and engage with our community.

**Comment by ozlemo5356-alt** [comment]
Hello, my name is Özlem,
I have just purchased your book SOCS, and have just started to read it,
**in p-1-17, reference 4, Soderlind G,2017**.... ----> can it be wrong? 
Best
